plugins/org.jkiss.dbeaver.erd.ui/src/org/jkiss/dbeaver/erd/ui/part/AttributePart.java

```diff
@@ -289,4 +292,21 @@ public ConnectionAnchor getTargetConnectionAnchor(ConnectionEditPart connection)
     public ConnectionAnchor getTargetConnectionAnchor(Request request) {
         return new ChopboxAnchor(getFigure());
     }
+
+    @Override
+    protected AccessibleEditPart getAccessibleEditPart() {
+        if (this.accPart == null) {
+            this.accPart = new AccessibleGraphicalEditPart() {
+                public void getName(AccessibleEvent e) {
+                    e.result = NLS.bind(ERDUIMessages.erd_accessibility_attribute_part,ERDUIUtils.getFullAttributeLabel(getDiagram(), getAttribute(),
+                        true, true));
+                }
+
+                public void getDescription(AccessibleEvent e) {
+                }
```

Comment:
I didn't mean to remove assigning null or empty string to e.result and leave a method with empty body.
Please, answer the question: can we remove this method or not?
Of we can't remove it, it's better to add assigning of an empty string or comment.